pkg/rpc/response/errors.go

```diff
@@ -20,6 +20,18 @@ type (
 var (
 	// ErrInvalidParams represents a generic 'invalid parameters' error.
 	ErrInvalidParams = NewInvalidParamsError("", nil)
+	// ErrAlreadyExists represents SubmitBlock error with code -501
+	ErrAlreadyExists = NewSubmitBlockError(-501, "Block or transaction already exists and cannot be sent repeatedly.")
```

Comment:
It's not just `SubmitBlock`, `sendrawtransaction` uses the same error answers.

Reply:
Changed `SubmitBlockError` to `SubmitError` and added these errors to sendrawtransaction